planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/src/utilities.cpp

```diff
@@ -278,8 +280,8 @@ PredictedPath resamplePredictedPath(
 {
   PredictedPath resampled_path;
 
-  rclcpp::Duration t_delta = rclcpp::Duration::from_seconds(resolution);
-  rclcpp::Duration prediction_duration = rclcpp::Duration::from_seconds(duration);
+  rclcpp::Duration t_delta(resolution);
```

Comment:
I guess it's better to use from_seconds function.

Reply:
Fixed.